kobo/apps/stripe/views.py

```diff
@@ -82,22 +74,18 @@ def start_checkout_session(customer_id, price_id, organization_uid):
             payment_method_types=["card"],
             success_url=f'{settings.KOBOFORM_URL}/#/plans?checkout_complete=true',
         )
-        return session
 
-    def get(self, request):
+    def post(self, request):
         serializer = CheckoutLinkSerializer(data=request.query_params)
         serializer.is_valid(raise_exception=True)
         serializer_data = serializer.validated_data
         price_id = serializer_data['price_id']
         organization_uid = serializer.validated_data.get('organization_uid')
-        try:
-            Price.objects.get(
-                active=True,
-                product__active=True,
-                id=price_id
-            )
-        except ObjectDoesNotExist:
-            return Response({'status': 'Price not found or inactive'}, status=status.HTTP_404_NOT_FOUND)
+        Price.objects.get(
```

Comment:
Is the idea here that we actively want it to error when no price exists? I don't think I have an opinion between a GET with 404 vs POST. I'm not sure I like this uncaught DoesNotExist possibility. An end user could not understand our API and send random price ids that don't exist. That should generate a 4xx level error. I could see some justification to log a warning to help get our attention but I don't see it as an error if the end user of the API can generate it.

It could be solved by adjusting the serializer to actually reference the Price by id instead of just a CharField. Unsure if this would make the code easier to read or not.

Not a blocker but let's change or discuss this.